homeassistant/components/climate/knx.py

```diff
@@ -48,9 +48,9 @@
                  default=DEFAULT_SETPOINT_SHIFT_STEP): vol.All(
                      float, vol.Range(min=0, max=2)),
     vol.Optional(CONF_SETPOINT_SHIFT_MAX):
-        vol.All(int, vol.Range(min=-32, max=0)),
```

Comment:
Can you move the defaults back into the schema? I moved them to the setup and pass then in as second argument to get because defaults weren't valid values